src/components/inspectors/TimerExpression.vue

```diff
@@ -327,31 +245,12 @@ export default {
       this.periodicity = 'week';
       this.endDate = DateTime.local().toUTC().toISO();
       this.ends = 'never';
-      this.weekdays.forEach(weekday => weekday.selected = false);
+      this.expression = null;
+      this.selectedWeekdays = [];
     },
     clickWeekDay(weekday) {
       weekday.selected = !weekday.selected;
     },
```

Comment:
This method doesn't appear to be used here anymore.

Reply:
Weird. It didn't grey out the method for me in phpstorm.